wd_helper.au3

```diff
@@ -1722,7 +1722,6 @@ EndFunc   ;==>_WD_SetElementValue
 ; Parameters ....: $sSession   - Session ID from _WD_CreateSession
 ;                  $sElement   - Element ID from _WD_FindElement
 ;                  $sCommand   - one of the following actions:
-;                  |
```

Comment:
This line was there by design. It provided some separation from the prior line, which carries over to the help file as well.

Reply:
reverted